package.json

```diff
@@ -153,10 +158,13 @@
     "setupTestFrameworkScriptFile": "<rootDir>/tests/setup.js",
     "testURL": "http://localhost:8065"
   },
+  "jest-junit": {
+    "output": "build/test-results.xml"
+  },
   "scripts": {
     "check": "eslint --ignore-pattern node_modules --ignore-pattern non_npm_dependencies --ignore-pattern dist --ext .js --ext .jsx . --quiet",
     "fix": "eslint --ignore-pattern node_modules --ignore-pattern non_npm_dependencies --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix",
-    "build": "cross-env NODE_ENV=production webpack --display-error-details --verbose --progress",
+    "build": "cross-env NODE_ENV=production webpack --display-error-details --verbose",
```

Comment:
Dropped the `--progress` flag here. This was responsible for the vast majority of the build output, since webpack would emit useless progress indicators in that context. I'm assuming this isn't as important for the `build` target as it is when running manually (e.g. via a human).

If we wanted to keep it here, we could hoist this command right out and into the `Jenkinsfile` if needed.